components/Counter/index.js

```diff
@@ -0,0 +1,14 @@
+import { formatDistanceToNow } from "date-fns";
+
+export default function Counter({ startDate, endDate }) {
+  const date = new Date();
+  if (date < new Date(startDate)) {
+    const countdown = formatDistanceToNow(new Date(startDate));
+    return <p>in {countdown}</p>;
+  } else if (date >= new Date(startDate) && date <= new Date(endDate)) {
+    return <p>Ongoing</p>;
+  } else {
+    const countdown = formatDistanceToNow(new Date(endDate), date);
+    return <p>{countdown} ago</p>;
+  }
```

Comment:
You don't need the `else`s, since you are retuning, do just:

```js
export default function Counter({ startDate, endDate }) {
  const now = new Date();
  const start = new Date(startDate);
  if (now < start) {
    const countdown = formatDistanceToNow(start);
    return <p>in {countdown}</p>;
  }
  const end = new Date(endDate);
  if (now >= start && now <= end) {
    return <p>Ongoing</p>;
  }
  const countdown = formatDistanceToNow(end, now);
  return <p>{countdown} ago</p>;
}
```

(called the dates `now`, `start`, `end` as well)